pytorch_lightning/core/lightning.py

```diff
@@ -1966,6 +1965,23 @@ def model_size(self) -> float:
             )
         return get_model_size_mb(self)
 
+    @property
+    def use_amp(self) -> bool:
```

Comment:
properties need to be added here so theyre not exported to torchscript: https://github.com/PyTorchLightning/pytorch-lightning/blob/7ee690758ccad7f702460d056f6369c1d4371a46/pytorch_lightning/core/lightning.py#L69-L87

Reply:
Yes @daniellepintz I ran into this too a while ago: #10092

Reply:
Correct. It's bad.

You could try pushing https://github.com/pytorch/pytorch/issues/67146 internally.